src/modules/Verifications/components/Atribute/useAttribute.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { OptionType, VerificationAttribute } from 'src/core/adaptors';
+
+export const useAttribute = (
+  index: number,
+  onChangeAttribute: (index: number, attribute?: OptionType, operator?: OptionType, value?: string) => void,
+  attribute?: VerificationAttribute,
+) => {
+  const operators: OptionType[] = [
+    { value: 'EQUAL', label: 'Is equal to' },
+    { value: 'NOT', label: 'Is not' },
+    { value: 'BIGGER', label: 'Is greater than' },
+    { value: 'SMALLER', label: 'Is less than' },
+  ];
+
+  const [selected, setSelected] = useState<OptionType>({ label: attribute?.name || '', value: attribute?.id || '' });
+  const [selectedOperator, setSelectedOperator] = useState<OptionType>({
+    value: attribute?.operator || '',
+    label: attribute ? operators.find(item => item.value === attribute.operator)?.label || '' : '',
```

Comment:
more readable in this way:
`operators?.find(item => item.value === attribute.operator)?.label || ''`

Reply:
I cannot remove the condition of checking attribute. it is practical to distinguish if user is creating a new verification or updating an old one. It also causes type script error
For example if user is creating a new verification, attribute is empty, so the find operation shouldn't execute